_eap-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
+Subproject commit d9281fa6c7ca7a498bdf95049c64bbbe41b989cf
```

Comment:
I agree we need to update the EAP quickstarts also. Did you intend to include it in this PR? I think it's safe to do so, but we need to be mindful of it.